### PR TITLE
Move the `divide` plugin into builtins

### PR DESCRIPTION
**Pull request by manushak**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- Move the sum plugin from if-plugins into IF builtins
- update the manifest file

### CODE REVIEW

File: src/builtins/divide/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      inputs:
+        - timestamp: 2023-08-06T00:00
+          duration: 3600
+          vcpus-allocated: 24
+```
+
+You can run this example by saving it as `./examples/manifests/divide.yml` and executing the following command from the project root:
+
+```sh
+if --manifest ./examples/manifests/divide.yml --output ./examples/outputs/divide.yml
```
Comment (narekhovhannisyan) [inline src/builtins/divide/README.md:91]: if or ie?

File: src/builtins/divide/index.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    input: PluginParams,
+    numerator: string,
+    denominator: number | string
+  ) => {
+    const schema = z
+      .object({
+        [numerator]: z.number(),
+        [denominator]: z.number().optional(),
+      })
+      .refine(_data => {
```
Comment (narekhovhannisyan) [inline src/builtins/divide/index.ts:66]: please do something  with the warning

**Comment by jmcook1186** [comment]
@jmcook1186 to review

File: src/builtins/divide/README.md (hunk 1 above)
Comment (manushak) [reply src/builtins/divide/README.md]: fixed

**Review by jmcook1186 — APPROVED** [review]
updated import paths in manifests
now sgtm

**Review by MariamKhalatova — APPROVED** [review]

**Pull request merged**